frontend/packages/dev-console/integration-tests/features/addFlow/helm-chart-repositories.feature

```diff
@@ -38,4 +38,14 @@ Feature: Install the Helm Release
               And user enters Description as "test"
               And user enters URL as "https://raw.githubusercontent.com/IBM/charts/master/repo/community/index.yaml"
               And user clicks on Create button
-             Then user can see "helm-test1" under Chart Repositories in Helm Charts catalog page
+               Then user can see "helm-test1" for resource "helm-test1" and type "projecthelmchartrepository" under Chart Repositories in Helm Charts catalog page
+
+         @regression
```

Comment:
Can you add epic tag as well
```suggestion
         @regression @odc-6685
```

Reply:
@sanketpathak but the form was introduced in ODC-6452

Reply:
It's related to the test scenario created with the epic